roles/openshift_logging_elasticsearch/tasks/main.yaml

```diff
@@ -19,6 +19,16 @@
     msg: Invalid deployment type, one of ['data-master', 'data-client', 'master', 'client'] allowed
   when: not openshift_logging_elasticsearch_deployment_type in __allowed_es_types
 
+- when:
+  - openshift_logging_elasticsearch_storage_type == "hostmount"
+  name: "Ensure openshift_logging_elasticsearch_hostmount_path is set"
+  assert:
+    that:
+    - openshift_logging_elasticsearch_hostmount_path is defined
+    - "openshift_logging_elasticsearch_hostmount_path != ''"
+    msg: |-
+      openshift_logging_elasticsearch_hostmount_path has not been set and is required for openshift_logging_elasticsearch_storage_type = "hostmount"
```

Comment:
The bz claims this used to work:
```
openshift_logging_elasticsearch_storage_type=hostmount
```
but no `openshift_logging_elasticsearch_hostmount_path` is set.  How did this work in previous releases?  Is there some default setting for `openshift_logging_elasticsearch_hostmount_path`?  What will happen if adding the `assert` somehow introduces a regression?

Reply:
before we didn't support setting a hostmount for a new deployment so this would have been updated after installation. We supported maintaining any hostmounts configured when doing an upgrade. 

https://github.com/openshift/openshift-ansible/pull/11771/files#diff-7e527e8dd0e0e7c99f51a7a31f35bf70R96

https://github.com/openshift/openshift-ansible/pull/11771/files#diff-7e527e8dd0e0e7c99f51a7a31f35bf70R169